packaging/rpm/flightctl.spec

```diff
@@ -1,6 +1,9 @@
 %define debug_package %{nil}
 
 Name: flightctl
+# PLEASE READ BEFORE CHANGING the Version: field
+# https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/
+# TL;DR With upstream releases 0.4.0, 0.4.1, 0.5.0-rc1, 0.5.0-rc2, 0.5.0, the two "release candidates" should use 0.5.0~rc1 and 0.5.0~rc2 in the Version: field
```

Comment:
This is handled by packit here really: https://github.com/flightctl/flightctl/blob/main/.packit.yaml#L22

We normally don't keep the .spec up to date in terms of versions, etc.

I wonder if we need to use any packit flag or report a bug to packit

Reply:
see https://packit.dev/docs/pre-releases